server/controllers/getArtistPaints.js

```diff
@@ -0,0 +1,27 @@
+const { getArtistPaintsQuery } = require('../database/queries');
+
+const getArtistPaints = async (req, res, next) => {
+  try {
+    const { artistId } = req.params;
+    if (artistId > 0) {
+      const { rows } = await getArtistPaintsQuery(artistId);
+      if (rows.length !== 0) {
+        res.json({ statusCode: 200, data: rows });
+      } else {
+        res.status(404).json({
+          statusCode: 404,
+          message: "Sorry There's no artist for this id",
```

Comment:
There's no painting for this artist :smiley:

Reply:
You are right, thank you *.*